geometry_msgs/Pose pose
float64 quality
float64 width
uint32 image_u
uint32 image_v
